site/content/docs/Installation/_index.md

```diff
@@ -244,6 +244,7 @@ In order to run Kopia in a container, you must:
 * (recommended) mount `/app/cache` directory in which Kopia will be keeping a cache of downloaded data
 * (optional) mount `/app/logs` directory in which Kopia will be writing logs
 * mount any data directory used for locally-attached repository
+* mount `/app/rclone` directory in which RClone will look for `rclone.conf` file
```

Comment:
optionally, no? this should only be required when one is using an rclone backend, right?

Reply:
done